modules/cudf/src/column/copying.cpp

```diff
@@ -21,12 +21,13 @@
 
 namespace nv {
 
-ObjectUnwrap<Column> Column::gather(Column const& gather_map,
-                                    cudf::out_of_bounds_policy bounds_policy,
-                                    rmm::mr::device_memory_resource* mr) const {
-  auto result = cudf::gather(cudf::table_view{{*this}}, gather_map, bounds_policy, mr);
-  std::vector<std::unique_ptr<cudf::column>> contents = result->release();
-  return Column::New(std::move(contents[0]));
+Column::wrapper_t Column::gather(Column const& gather_map,
+                                 cudf::out_of_bounds_policy bounds_policy,
+                                 rmm::mr::device_memory_resource* mr) const {
+  return Column::New(
+    Env(),
+    std::move(
+      cudf::gather(cudf::table_view{{*this}}, gather_map, bounds_policy, mr)->release()[0]));
```

Comment:
What's the rationale for combining things into one giant expression instead of smaller individual steps?

Reply:
Just trying to make things more consistent while I was find-replacing all the `ObjectUnwrap` return signatures. We can make things more consistent the other way, but I like using temporaries instead of lvalues.

Reply:
It just gets difficult to visually parse what is what when there are nested function calls, derefs and array indexes in a single expression